src/libsync/syncfilestatustracker.cpp

```diff
@@ -157,6 +162,20 @@ SyncFileStatus SyncFileStatusTracker::fileStatus(const QString& systemFileName)
     return SyncFileStatus();
 }
 
+void SyncFileStatusTracker::slotPathTouched(const QString& systemFileName)
+{
+    QString fileName = systemFileName.normalized(QString::NormalizationForm_C);
+
+    QString folderPath = _syncEngine->localPath();
+    if (!fileName.startsWith(folderPath))
```

Comment:
I'm uncomfortable about Q_ASSERTing there, what if due to some reason the paths don't match exactly? missing drive letter, / vs \ or something like that? It's indicative of a bug, yes. Do we ship binaries with Q_ASSERT enabled?